app/src/main/res/drawable/background_pinentry_bruteforcelock_timer.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Can you verify this with the designs? They seem to specify a corner radius for this shape as well.

Reply:
Corner radius? Do you see it on zeplin? Because I cannot. 🤔

Reply:
@vinaysshenoy is it okay if I make these changes in my next PR? The feature is behind a flag anyway.

Reply:
Yup.